weldx/__init__.py

```diff
@@ -14,7 +14,7 @@
     )
 
 # main modules
-import weldx.utility  # import this first to avoid circular dependencies
+import weldx.util  # import this first to avoid circular dependencies
```

Comment:
Just a general comment: Should test if this is still the case.

Reply:
I've changed it along the way to revert it back. There is still a circular dep within the util module